students/ryba_paula/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,5 @@
+n = int(input())
```

Comment:
nazwy zmiennych powinny byc bardziej opisowe, prezentować jasną infomacje o tym co to jest. Pojedyńcze literki nie zawsze są czytelne, zwłaszcza, że w tym zadania mamy jabłka i studentów.
Kwestia nazwy zmienny tyczy się wszystkich zadań.

Reply:
No ja to rozumiem, aczkolwiek Snakify twierdziło "The program reads the numbers N and K" - zatem myślałam, że tak ma być :)